web/components/issues/issue-layouts/kanban/properties.tsx

```diff
@@ -166,7 +166,11 @@ export const KanBanProperties: React.FC<IKanBanProperties> = observer((props) =>
       {/* sub-issues */}
       {displayProperties && displayProperties?.sub_issue_count && !!issue?.sub_issues_count && (
         <Tooltip tooltipHeading="Sub-issues" tooltipContent={`${issue.sub_issues_count}`}>
-          <div className="flex h-5 flex-shrink-0 items-center justify-center gap-2 overflow-hidden rounded border-[0.5px] border-custom-border-300 px-2.5 py-1">
+          <div
+            className={`flex h-5 flex-shrink-0 items-center justify-center gap-2 overflow-hidden rounded border-[0.5px] border-custom-border-300 px-2.5 py-1 ${
+              isReadOnly ? "cursor-not-allowed" : "cursor-pointer"
```

Comment:
Why to disable this? It's anyways not clickable

Reply:
Oh okay. In that case we should not even show it as `cursor-pointer` cause then it may seem as if it is clickable?

Reply:
Or we can keep it as we do have a tooltip. Not sure what's the best UX.

Reply:
Remove the `cursor-pointer`, it should only be added to actionable fields. Do the same everywhere.

Reply:
Okk